includes/polling/functions.inc.php

```diff
@@ -247,7 +247,14 @@ function poll_device($device, $options)
             }
         }
         foreach ($config['poller_modules'] as $module => $module_status) {
-            if ($force_module === true || $attribs['poll_'.$module] || ( $module_status && !isset($attribs['poll_'.$module]))) {
+            $os_module_status = $config['os'][$device['os']]['poller_modules'][$module];
+            d_echo("Global module enabled: ".$module_status. "\n");
```

Comment:
This is too verbose, if you can condense to one line that would be great.
"Enabled G+ O- D+" or something like that.  or maybe `c_echo("Enabled %gG,%rO,%gD, $debug);`